financialaid/management/commands/generate_exchange_rates.py

```diff
@@ -0,0 +1,16 @@
+"""
+Generates CurrencyExchangeRate objects
+"""
+from django.core.management import BaseCommand
+
+from financialaid.tasks import sync_currency_exchange_rates
+
+
+class Command(BaseCommand):
+    """
+    Generates CurrencyExchangeRate objects by calling sync_currency_exchange_rates
+    """
+    help = "Generates CurrencyExchangeRate objects by calling sync_currency_exchange_rates"
```

Comment:
To an operations person running this management command, who probably does _not_ have intimate knowledge of this codebase, this description is not helpful. Instead, I would suggest: "Update the local database with the latest exchange rate information from the Open Exchange Rates API (openexchangerates.org)"